src/py-opentimelineio/CMakeLists.txt

```diff
@@ -1,2 +1,21 @@
+#------------------------------------------------------------------------------
+# py-opentimelineio/CMakeLists.txt
+
 add_subdirectory(opentime-bindings)
 add_subdirectory(opentimelineio-bindings)
+
+# Install pure-python OTIO packages to match PyPI wheel structure
+install(DIRECTORY "${PROJECT_SOURCE_DIR}/src/py-opentimelineio/opentimelineio/"
+    DESTINATION "${OTIO_RESOLVED_PYTHON_INSTALL_DIR}/opentimelineio")
+
```

Comment:
```suggestion
if(OTIO_PYTHON_INSTALL)
    install(DIRECTORY "${PROJECT_SOURCE_DIR}/src/py-opentimelineio/opentimelineio/"
        DESTINATION "${OTIO_RESOLVED_PYTHON_INSTALL_DIR}/opentimelineio")
endif()
```

Reply:
Did you mean to guard this new bit, as I've suggested above?

Reply:
This wasn't needed because this module is guarded in the root CMake by  `OTIO_PYTHON_INSTALL`, so all three of these install statements are implicitly guarded by it:

```
if(OTIO_PYTHON_INSTALL)
    add_subdirectory(src/py-opentimelineio)
endif()
```

Reply:
ah gotcha, forget to expand the context